### PR TITLE
GenioWindow: cleanup/refactor

### PR DESCRIPTION
**Pull request by jackburton79**

### CODE REVIEW

File: src/ui/EditorTabManager.cpp [hunk 1]
```diff
@@ -55,13 +54,34 @@ EditorTabManager::EditorBy(entry_ref* ref)
...
+		Editor* editor = EditorAt(index);
+		if (editor == nullptr) {
+			BString notification;
+			notification
+				<< "Index " << index << ": NULL editor pointer";
+			LogInfo(notification.String());
+			continue;
+		}
+
+		if (editor->NodeRef() == nodeRef)
```
Comment (Freaxed) [inline src/ui/EditorTabManager.cpp:78]: Shouldn't we compare by value here?

Comment (jackburton79) [reply src/ui/EditorTabManager.cpp:78]: That's a doubt I had, in fact

Comment (Freaxed) [reply src/ui/EditorTabManager.cpp:78]: To me yes, we should. We could also change the signature function to use a "const node_ref&" to avoid confusion.

Comment (jackburton79) [reply src/ui/EditorTabManager.cpp:78]: Wouldn't be a problem if we start supporting "empty" files ?

Comment (jackburton79) [reply src/ui/EditorTabManager.cpp:78]: Oh sorry, I got it wrong: you meant a node_ref& only for the EditorBy(), right ?

Comment (jackburton79) [reply src/ui/EditorTabManager.cpp]: Now I compare the values.
I didn't change the method signature yet, though

**Comment by Freaxed** [comment]
OK, fine with me!

**Pull request merged**